src/platform/src/main/java/org/geoserver/platform/ModuleStatusImpl.java

```diff
@@ -51,6 +51,8 @@
 
     private boolean isAvailable;
 
+    private List<ModuleCapabilities.Capability> capabilities;
```

Comment:
Taking into account my comment above, here you would have a list of ``ModuleCapability``.

Reply:
fixed